ooniprobe/Utility/UIDevice-Hardware.m

```diff
@@ -0,0 +1,31 @@
+#include <sys/sysctl.h>
+#import "UIDevice-Hardware.h"
+
+@interface UIDevice (Hardward)
+
+- (NSString *)modelNameForModelIdentifier:(NSString *)modelIdentifier;
+
+@end
+
+@implementation UIDevice (Hardware)
+
+- (NSString *)getSysInfoByName:(char *)typeSpecifier
+{
+    size_t size;
+    sysctlbyname(typeSpecifier, NULL, &size, NULL, 0);
+
+    char *answer = malloc(size);
+    sysctlbyname(typeSpecifier, answer, &size, NULL, 0);
+
+    NSString *results = [NSString stringWithCString:answer encoding: NSUTF8StringEncoding];
+
+    free(answer);
+    return results;
+}
+
+- (NSString *)modelIdentifier
+{
+    return [self getSysInfoByName:"hw.machine"];
+}
+
+@end
```

Comment:
Do we still need this code since we are dropping the dependency?

Is it maybe not possible to call the `getSysInfoByName` method in the function creating the email directly?

Reply:
This comment may also due to my lack of understanding how objective-c patterns work, but perhaps it's sufficient to just define a function called `getSysInfoByName` inside of `SettingsCategoriesTableViewController.m`.

Reply:
Yes we still need this code as we need to extend the UIDevice class to be able to call that function.
Also `getSysInfoByName` doesn't belong to a ViewController, let's keep the code clean and in order.

Reply:
this is one of the reason why I would have kept the library in the first place, one less class to worry about in the main project

Reply:
Ok understood. Then let's perhaps call it a name different than `UIDevice-Hardware` so that we avoid confusing this with the library we have dropped called `UIDevice-Hardware` (https://github.com/erichoracek/UIDevice-Hardware)?

Reply:
Any idea for a new name? UIDevice-Extended?

Reply:
`DeviceInfo`?

Reply:
Good call

Reply:
Can we resolve this convo, @lorenzoPrimi ?

Reply:
Sure, it has been renamed